collect_app/src/test/java/org/odk/collect/android/database/InstanceDatabaseMigratorTest.kt

```diff
@@ -107,4 +141,20 @@ class InstanceDatabaseMigratorTest {
             put(GEOMETRY_TYPE, "GeometryType")
         }
     }
+
+    private fun getContentValuesForInstanceV7(): ContentValues {
```

Comment:
There is no difference between `getContentValuesForInstanceV6` and `getContentValuesForInstanceV7`. You should add your new column to `getContentValuesForInstanceV7`.